packages/device-connector/src/deployment-server/service.ts

```diff
@@ -81,16 +93,14 @@ export const fetchDeviceType = async (typeId: string): Promise<any> => {
     }
   })
 
-  return await resp.json()
+  return await resp.json() as DeviceType
 }
 
-export const deleteDeviceRequest = async (deviceId: string): Promise<any> => {
+export const deleteDeviceRequest = async (deviceId: string): Promise<void> => {
   const address = env.SERVER_URI != null ? env.SERVER_URI : '127.0.0.1:3001'
   const url = `http://${address}/devices/${deviceId}`
```

Comment:
We may want to be able configure another protocol (HTTPS)

Reply:
Yeah, I am aware of this and it is on my ToDo list for changes in the near future (see Trello)